### DISCUSSION TITLE
To know about navbar features

### DISCUSSION THREAD

**Discussion by vikas9616** [saadeghi/daisyui#3041] [Q&A] [answered]
how we can display ghost on the right side of screen in small devices.
![Screenshot 2024-05-18 114854](https://github.com/saadeghi/daisyui/assets/117437513/479eb6f7-3d7a-4c59-9dbd-67baf7775312)
please answer

**Comment by melvinczyk** [comment, marked as answer]
In your nav bar you can separate it into start, center, and end depending on where you want the feature to be. You can add the ghost button to the end section for it to be on the right:
```
<div class="navbar bg-base-300>
    <div class="navbar-start"></div>
    <div class="navbar-center"></div>
    <div class="navbar-end">
        <!-- Add content here -->
    </div>
</div>
```
